src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/SlhDsa/SlhDsaContractTests.cs

```diff
@@ -283,6 +410,151 @@ public static void VerifyData_ByteArray_CallsCore(SlhDsaAlgorithm algorithm)
             Assert.Equal(2, slhDsa.VerifyDataCoreCallCount);
         }
 
+        [Theory]
+        [InlineData(SlhDsaTestHelpers.Md5Oid, 128 / 8)]
+        [InlineData(SlhDsaTestHelpers.Sha1Oid, 160 / 8)]
+        [InlineData(SlhDsaTestHelpers.Sha256Oid, 256 / 8)]
+        [InlineData(SlhDsaTestHelpers.Sha384Oid, 384 / 8)]
+        [InlineData(SlhDsaTestHelpers.Sha512Oid, 512 / 8)]
+        [InlineData(SlhDsaTestHelpers.Sha3_256Oid, 256 / 8)]
+        [InlineData(SlhDsaTestHelpers.Sha3_384Oid, 384 / 8)]
+        [InlineData(SlhDsaTestHelpers.Sha3_512Oid, 512 / 8)]
+        [InlineData(SlhDsaTestHelpers.Shake128Oid, 256 / 8)]
+        [InlineData(SlhDsaTestHelpers.Shake256Oid, 512 / 8)]
+        [InlineData("1.0", 0)]
+        [InlineData("1.0", 1)]
+        [InlineData("1.0", 2)]
+        public static void SignPreHash_CallsCore(string hashAlgorithmOid, int outputBytes)
```

Comment:
`outputBytes` as a parameter reads to me as if you expect that to be the length of the signature.  `hashLength`, maybe?